dss/events/handlers/index.py

```diff
@@ -126,25 +132,110 @@ def add_index_data_to_elasticsearch(bundle_key, index_data, logger) -> None:
 
 
 def create_elasticsearch_index(logger):
+    index_mapping = {
+        "mappings": {
+            "query": {
+                "properties": {
+                    "query": {
+                        "type": "percolator"
+                    }
+                }
+            }
+        }
+    }
     try:
         es_client = ElasticsearchClient.get(logger)
         response = es_client.indices.exists(DSS_ELASTICSEARCH_INDEX_NAME)
-        if response is False:
+        if not response:
             logger.debug(f"Creating new Elasticsearch index: {DSS_ELASTICSEARCH_INDEX_NAME}")
-            response = es_client.indices.create(DSS_ELASTICSEARCH_INDEX_NAME, body=None)
+            response = es_client.indices.create(DSS_ELASTICSEARCH_INDEX_NAME, body=index_mapping)
             logger.debug(f"Index creation response: {json.dumps(response, indent=4)}")
         else:
             logger.debug(f"Using existing Elasticsearch index: {DSS_ELASTICSEARCH_INDEX_NAME}", )
     except Exception as ex:
         logger.critical(f"Unable to create index {DSS_ELASTICSEARCH_INDEX_NAME}  Exception: {ex}")
+        raise
 
 
 def add_data_to_elasticsearch(bundle_id, index_data, logger) -> None:
     try:
         ElasticsearchClient.get(logger).index(index=DSS_ELASTICSEARCH_INDEX_NAME,
                                               doc_type=DSS_ELASTICSEARCH_DOC_TYPE,
                                               id=bundle_id,
-                                              body=json.dumps(index_data, indent=4))
-
+                                              body=json.dumps(index_data),
+                                              refresh=True)
```

Comment:
Using refresh when adding index docs at scale is too expensive. It is also unnecessary and has been removed.